src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -62,6 +62,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('no_push', help="Indicates whether the image built should be pushed to the registry.", arg_type=get_three_state_flag())
         c.argument('no_logs', help="Do not show logs after successfully queuing the build.", action='store_true')
 
+    with self.argument_context('acr import') as c:
+        c.argument('registry_name', options_list=['--registry', '-r'])
+        c.argument('resource_id', help='The ARM resource ID of the source container registry.')
+        c.argument('source', help='A fully qualified source indentifier (e.g. registry.azurecr.io/repository:tag for an image identifier).')
+        c.argument('target_tags', options_list=['--target_tags', '-t'], help="The repository and optionally a tag in the 'repository:tag' format for images.", action='append')
```

Comment:
@djyou  why are we using _ for parameter name seperators shouldn't this be --target-tags 
Either way does not align with --image which is what build uses and I think we need to have the same convention here.